xml/System.Windows.Forms/AccessibleEvents.xml

```diff
@@ -616,6 +616,12 @@
       <MemberValue>15</MemberValue>
       <Docs>
         <summary>An application is about to exit drag-and-drop mode. Applications that support drag-and-drop operations must send this event; the system does not.</summary>
+        <remarks>
```

Comment:
@RussKie are these remarks information that is important to customer to know? If so, we need to move them to a different place because enum fields don't have remarks on docs
https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.accessibleevents?view=netcore-3.0

Just let me know so I can work on the PR if needed.